Sources/Gtk/Widgets/Paned.swift

```diff
@@ -0,0 +1,49 @@
+import CGtk
+
+open class Paned: Widget, Orientable {
+    public init(orientation: Orientation) {
+        super.init()
+        widgetPointer = gtk_paned_new(orientation.toGtkOrientation())
+    }
+
+    public var startChild: Widget? {
+        didSet {
+            gtk_paned_set_start_child(opaquePointer, startChild?.castedPointer())
```

Comment:
hmm do we need to unset the parent of the previous child widget?

Reply:
> hmm do we need to unset the parent of the previous child widget?

I think so, to keep the Swift side of the Gtk view hierarchy up to date with the C side